app/src/main/res/layout/insurance_contract_card.xml

```diff
@@ -40,11 +40,12 @@
             tools:visibility="visible" />
 
         <ImageView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
+            android:layout_width="@dimen/base_margin_triple"
```

Comment:
I think we should introduce a separate `dimen` for this, else we're going to have weird behaviours if we decide to switch around `base_margin`s 😬